postgres-protocol/src/message/frontend.rs

```diff
@@ -132,14 +132,19 @@ pub fn close(variant: u8, name: &str, buf: &mut BytesMut) -> io::Result<()> {
     })
 }
 
-pub struct CopyData<T> {
+pub struct CopyData<T> 
+where T: Sync
+{
     buf: T,
     len: i32,
 }
 
+
+unsafe impl<T> Sync for CopyData<T> where T: Sync {}
```

Comment:
This is exactly equivalent to the automatic implementation of Sync for CopyData.

Reply:
True, will be fixed